docs/wagtail-pages.md

```diff
@@ -0,0 +1,207 @@
+# Wagtail pages
+
+[Wagtail pages](http://docs.wagtail.io/en/v1.13.4/topics/pages.html) are 
+[Django models](https://docs.djangoproject.com/en/1.11/topics/db/models/) 
+that are constructed of 
+[fields](#fields), [StreamFields](#streamfields), and [panels](#panels) 
+that are rendered in a standard way. 
+All CFPB Wagtail pages should inherit from the 
+[`v1.models.base.CFGOVPage` class](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/base.py#L60).
+
+There are types of information defined on a new Wagtail page model: 
+basic [database fields](#fields) (like any Django model), 
+specialized database fields called [StreamFields](#streamfields) that allow for freeform page content, 
+and [editor panels](#panels) that present these fields to content editors. 
+
+## Fields
+
+Database fields in Wagtail pages work exactly the same as in 
+[Django models](https://docs.djangoproject.com/en/1.11/topics/db/models/#fields), 
+and Wagtail pages can use any [Django model field](https://docs.djangoproject.com/en/1.11/ref/models/fields/). 
+
+For example, our 
+[`BrowsePage` includes a standard Django `BooleanField`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/browse_page.py#L54) 
+that allows content editors to toggle secondary navigation sibling pages:
+
+```python
+from django.db import models
+
+from v1.models.base import CFGOVPage
+
+
+class BrowsePage(CFGOVPage):
+    secondary_nav_exclude_sibling_pages = models.BooleanField(default=False)
+```
+
+## StreamFields
+
+StreamFields are special Django model fields provided by Wagtail for 
+[freeform page content](https://docs.wagtail.io/en/v1.13.4/topics/streamfield.html). 
+They allow a content editor to pick any number number of optional components
+and place them in any order within their StreamField. 
+In practice this provides the 
+[flexibilty of a large rich text box with the structure of individual components](https://torchbox.com/blog/rich-text-fields-and-faster-horses/).
+
+For example, our [`LandingPage` page model includes a `header` StreamField](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/landing_page.py#L13) that can have a hero and/or a text introduction:
+
+```python
+from wagtail.wagtailcore.fields import StreamField
+
+from v1.atomic_elements import molecules
+from v1.models import CFGOVPage
+
+
+class LandingPage(CFGOVPage):
+
+    header = StreamField([
+        ('hero', molecules.Hero()),
+        ('text_introduction', molecules.TextIntroduction()),
+    ], blank=True)
+```
+
+**Insert link to StreamField docs here.**
+
+## Panels
+
+Editor panels define how the page's [fields](#fields) and [StreamFields](#streamfields) will be organized for content editors.
+
+The base Wagtail `Page` class and the [`v1.models.base.CFGOVPage` class](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/base.py#L60) 
+defines a specific set of panels to which all fields should be added:
```

Comment:
Is the term "panels" one that the average developer or editor would understand? I tend to think of these as "tabs" on the page edit screen.

Reply:
Yeah, I describe where each panel appears below, but maybe stating it here too would be good. I also thought about including screenshots, but that seemed a bit superfluous.